domains/pathak.json

```
@@ -0,0 +1,9 @@
+{
+   "owner": {
+      "username": "Mehulpathak21"
+      "email": "mehulpathak48@gmail.com"
+    },
+    "record": {
```

```suggestion
    "records": {
```
invalid Domain Structure please read -> https://docs.is-a.dev/domain-structure/